package.json

```diff
@@ -3,50 +3,50 @@
-    "server-dev": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-dev.js",
-    "server-hot": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-hot.js",
-    "server-wds-dev": "webpack-dev-server --config webpack.config.dev.js --progress --colors --port 2992",
-    "server-wds-hot": "webpack-dev-server --config webpack.config.hot.js --hot --progress --colors --port 2992 --inline",
-    "server-wds-test": "webpack-dev-server --port 3001 --config webpack.config.test.js --colors",
-    "sources": "http-server -p 3001 .",
-    "watch": "webpack --watch --colors",
-    "prod": "concurrent -kr 'npm run watch' 'npm run server' 'npm run sources'",
-    "dev": "concurrent -kr 'npm run server-wds-test' 'npm run server-wds-dev' 'npm run server-dev'",
-    "hot": "concurrent -kr 'npm run server-wds-test' 'npm run server-wds-hot' 'npm run server-hot'",
-    "build": "webpack",
-    "install-dev": "selenium-standalone install",
-    "postinstall": "node heroku/scripts/not-heroku.js || (node heroku/scripts/install.js && npm run build)"
+    "test-server":          "npm run test-server-unit && npm run test-server-rest",
+    "test-server-cov":      "npm run test-server-unit-cov && npm run test-server-rest-cov",
+    "test-func":            "mocha --opts test/func/mocha.opts test/func/spec",
+    "test-func-cov":        "istanbul cover --config .istanbul.func.yml _mocha -- --opts test/func/mocha.opts test/func/spec",
+    "test-func-dev":        "mocha --opts test/func/mocha.dev.opts test/func/spec",
+    "test":                 "npm run test-client && npm run test-server && npm run test-func",
+    "test-ci":              "npm run test-client-ci && npm run test-server-cov && npm run test-func-cov",
+    "test-ci-win":          "npm run test-client-ci-win && npm run test-server && echo 'TODO(36) fix Appveyor test-func'",
+    "test-cov":             "npm run test-client-cov && npm run test-server-cov && npm run test-func-cov",
+    "test-dev":             "npm run test-client-dev && npm run test-server && npm run test-func-dev",
+    "check":                "npm run lint && npm run test",
+    "check-ci":             "npm run lint && npm run test-ci",
+    "check-ci-win":         "npm run lint && npm run test-ci-win",
+    "check-cov":            "npm run lint && npm run test-cov",
+    "check-dev":            "npm run lint && npm run test-dev",
+    "start":                "node server/index.js",
+    "server":               "nodemon --watch client --watch server --watch templates --ext js,jsx server/index.js",
+    "server-dev":           "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-dev.js",
+    "server-hot":           "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-hot.js",
+    "server-wds-dev":       "NODE_ENV=development && webpack-dev-server --progress --colors --port 2992",
```

Comment:
All code in this repo needs to be windows / mac / linux compatible. This code won't work on windows.